### PR TITLE
aurvote: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/linux/aurvote.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# aurvote
+
+> Vote for packages in the Arch User Repository.
```
Comment (CleanMachine1) [inline pages/linux/aurvote.md:3]:
```suggestion
> Submit votes for packages in the Arch User Repository.
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/aurvote.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+- Vote for one or more AUR packages:
+
+`aurvote {{package1 package2 ...}}`
+
+- Unvote one or more AUR packages:
+
+`aurvote --unvote {{package1 package2 ...}}`
+
+- Check if one or ore AUR packages have already been voted:
```
Comment (blueskyson) [inline pages/linux/aurvote.md:19]:
```suggestion
- Check if one or more AUR packages have already been voted:
```

Haha.

File: pages/linux/aurvote.md (hunk 1 above)
Comment (marchersimon) [reply pages/linux/aurvote.md:3]: Why? I like my version.

Comment (CleanMachine1) [reply pages/linux/aurvote.md:3]: Then keep it.

This suggestion isn't really that meaningful tbh, feel free to resolve.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**